interpreter/exec/v128.ml

```diff
@@ -5,43 +5,43 @@ include Simd.Make
     let to_string s = s
 
     let to_i8x16 s =
-      List.map (fun i -> I8.of_bits (Int32.of_int (Bytes.get_int8 (Bytes.of_string s) i))) Simd.i8x16_indices
+      List.init 16 (fun i -> (Int32.of_int (Bytes.get_int8 (Bytes.of_string s) i)))
 
     let of_i8x16 fs =
       let b = Bytes.create bytewidth in
-      List.iter2 (fun i f -> Bytes.set_int8 b i (Int32.to_int (I8.to_bits f))) Simd.i8x16_indices fs;
+      let _ = List.init 16 (fun i -> Bytes.set_int8 b i (Int32.to_int (List.nth fs i))) in
```

Comment:
Ah, these ones you probably want to express as follows:
```suggestion
      List.iteri (fun i f -> Bytes.set_int8 b i (Int32.to_int f)) fs;
```

Reply:
Ah yes, that is what I need. Thanks!